### DISCUSSION TITLE
Apis for listening stats

### DISCUSSION THREAD

**Discussion by j-dexx** [tidal-music/discussions#11] [Ideas & Feature requests]
Would be nice to provide user listening stats so services like stats.fm could integrate.

**Comment by stefanmoro** [comment]
Hi @j-dexx, thanks for reaching out.

Could you please specify a bit closer what you mean with listening stats? Would you say that #10 already covers your question?

Thanks

**Reply by j-dexx** [reply]
Ah yes pretty much. Might be worth asking stats.fm what they would use they seem quite big for Spotify stats, currently have apple music in beta and asked about public APIs for tidal on twitter last year https://twitter.com/spotistats/status/1597739646402072576?t=kOH7oOaFqS-g_QyZ0YUKKw&s=19

**Reply by stefanmoro** [reply]
Thanks for the additional info.

I'll close this as a duplicate of #10 so that we get this additional info in that discussion as well.
